grace/models/classifier.py

```diff
@@ -69,18 +67,24 @@ def __init__(
         edge_output_classes: int = 2,
         num_heads: int = 1,
         dropout: float = 0.0,
-        verbose: bool = False,
     ):
         super(GNNModel, self).__init__()
 
         # Define the model attributes:
         self.classifier_type = classifier_type
+        self.input_channels = input_channels
+        self.hidden_graph_channels = hidden_graph_channels
+        self.hidden_dense_channels = hidden_dense_channels
+        self.node_output_classes = node_output_classes
+        self.edge_output_classes = edge_output_classes
+        self.num_heads = num_heads
+        self.dropout = dropout
+
         hidden_channels_list = [input_channels]
```

Comment:
isn't this a duplication of line 75? Do we need the two separate variables (hidden_channels_list and input_channels)?

Reply:
I suppose this assures that the `input_channels` (and `int` == num_features) is put into the list which will grow if other hidden layers are specified. Could be redesigned, agreement on that 😄